src/lasp_plumtree_broadcast_distribution_backend.erl

```diff
@@ -1005,11 +1005,11 @@ do(Function, Args) ->
 -endif.
 
 %% @private
-log_transmission(Term) ->
+log_transmission(Type, Payload, PeerCount) ->
     try
         case lasp_config:get(instrumentation, false) of
             true ->
-                ok = lasp_transmission_instrumentation:log(server, Term, node()),
+                ok = lasp_transmission_instrumentation:log(server, Type, Payload, PeerCount, node()),
```

Comment:
`lasp_transmission_instrumentation` stored something called `type` (that I renamed to `entity`). The possible values were: `client` and `server`.
Are you saying we don't need this distinction anymore?